File: Java/Car.java
class Car {
    Integger id;
    String license;
    Account driver;
    Integger passenger;

    public Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;
    }

    printDataCar(){
        System.out.printLn("License: " + license + " Driver: " + driver):
    }
}

File: PHP/car.php
<?php

require_once 'account.php';

class Car {
    public $id;
    public $license;
    public $driver;
    public $passenger;

    public function __constructor($license, 
            $driver) {
        $this->license = $license;
        $this->driver = $driver;
    }

    public function printDataCar() {
        echo "license: $license 
            Driver: $(this.name)";
    }

}